notificationdata.cpp

```diff
@@ -23,6 +23,15 @@ QString NotificationData::text() const {
 }
 
 QString NotificationData::img() const {
+    const QString imageSrc = imgSrc();
+    if(imageSrc != "")
```

Comment:
Use the isEmpty or the size methods instant compare operators. 
```suggestion
    if(imageSrc.size())
```